docs/versioning.md

```diff
@@ -0,0 +1,16 @@
+# Versioning
+
+Out of the box, the Pact Broker allows uploading of pact files with semver
+style versions (eg 2.0.1). For our usage, we wanted to be able to upload pact
+files from various branches in addition the released versions so that our
```

Comment:
```suggestion
files from various branches in addition to the released versions so that our
```

Reply:
Fixed in https://github.com/alphagov/govuk-pact-broker/pull/123/commits/54d61fbf7ff94d7d824e6ce31f4d6af4fc73cdfc.